### ISSUE TITLE
Two new buttons into right map: "copy coord." and "open this point in the external map"

### ISSUE THREAD

**Issue by paul-k-pastvu** [klimashkin/pastvu#118]
Currently, in the edit mode, there is an option to input coordinates in the decimal mode (and also copy them to using in the external sites):

![image](https://user-images.githubusercontent.com/60163100/72994724-b1b04080-3e08-11ea-863c-182558373dba.png)

This is available for "image owner" (user what an uploaded image), and also for moderators and administrators. But this is unavailable for other users and visitors.

Requested two new buttons into right map: 

1. "copy coordinates"
2. "open this point in the external map"

A proposed place for new buttons:

![image](https://user-images.githubusercontent.com/60163100/72996636-1c16b000-3e0c-11ea-8dcc-21e7b31acc42.png)

**Comment by paul-k-pastvu** [comment]
No! Not "Open in new page", but exactly "Open this _point_ in external map"!

**Comment by kabalin** [comment]
Эту задачу необходимо разделить на две. Кнопка показа координат это просто, можно классифицировать как "Good first issue. Кнопка открытия на внешней карте - задача средней сложности, для карты должен быть описан способ открытия, скажем ссылка с плейсхолдерами координат и зума может быть указана как параметр источника (наряду с максимальным зумом и прочих аттрибутов карты), если параметр с ссылкой для текущей присуствует - на карте показывается кнопка открывающая внешнюю карту с текущими координатами, если нет, то кнопка неактивна либо отсутвует. В идеале эта задача должна решаться после #279.
